src/components/SandpackExample.jsx

```diff
@@ -0,0 +1,32 @@
+import { Sandpack } from '@codesandbox/sandpack-react';
+
+export default function SandpackExample(htmlSample, cssSample, jsSample) {
+  return (
+    <section>
+      <Sandpack
+        template="vanilla"
+        files={{
+          '/index.html': {
+            code: htmlSample,
+            active: true,
+          },
+          '/index.js': jsSample,
+          'index.css': cssSample,
+          '/src/index.js': {
+            code: 'import "../index.css"; import "../index.js"',
+            hidden: true,
+          },
+        }}
+        customSetup={{
+          entry: '/accordion.html',
+        }}
+        options={{
+          showConsole: true,
+          showConsoleButton: true,
+          showLineNumbers: true,
+          wrapContent: true,
+        }}
+      />
+    </section>
+  );
+}
```

Comment:
It looks like we're not using this component anywhere. Can we go ahead and delete this file?

Reply:
Whoops. I must have forgotten to delete it. https://github.com/sparkbox/accessible-components/pull/105/commits/1fa508dd12b7bc45b33ba214d1f04fd1ea5dae89